File: src/main/java/dev/alexengrig/leetcode/_2_add_two_numbers/Task.java
package dev.alexengrig.leetcode._2_add_two_numbers;

interface Task {
    ListNode addTwoNumbers(ListNode l1, ListNode l2);

    class ListNode {
        int val;
        ListNode next;

        ListNode() {
        }

        ListNode(int val) {
            this.val = val;
        }

        ListNode(int val, ListNode next) {
            this.val = val;
            this.next = next;
        }
    }
}

File: src/main/python/160_intersection_of_two_linked_lists/task.py
from typing import Optional


class ListNode:
    def __init__(self, val=0, next: Optional["ListNode"] = None):
        self.val = val
        self.next = next


class Task:
    def getIntersectionNode(self, headA: ListNode, headB: ListNode) -> Optional[ListNode]:
        pass
